examples/schelling/model.py

```diff
@@ -70,7 +70,7 @@ def __init__(self, width=20, height=20, density=0.8, minority_pc=0.2, homophily=
                     agent_type = 0
 
                 agent = SchellingAgent((x, y), self, agent_type)
-                self.grid.position_agent(agent, (x, y))
+                self.grid.position_agent(agent, x, y)
```

Comment:
This is a bug where in the Schelling example, the number of agents in the scheduler does not match the number of agents in the space.